source/elastic_solver.py

```diff
@@ -234,8 +235,8 @@ def set_boundary_conditions(self, bcs):
         """
         assert isinstance(bcs, (list, tuple))
         # check format
-        # To Do: We don't know why this is not working.
         """
+        # To Do: We don't know why this is not working.
```

Comment:
Here again. We don't know why the format is wrong. It has something to do with the pressure bc.

Reply:
I guess that this is related to the previous comment...